docs/content/stable/yugabyte-platform/alerts-monitoring/prometheus-setup.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Use your own Prometheus instance
+headerTitle: Use a custom Prometheus instance
+linkTitle: Custom Prometheus
+description: Export universe metrics to your own Prometheus instance
+headcontent: Export universe metrics to your own Prometheus instance
+menu:
+  stable_yugabyte-platform:
+    parent: alerts-monitoring
+    identifier: prometheus-setup
+    weight: 90
+type: docs
+---
+
+YugabyteDB Anywhere provides [metrics](../anywhere-metrics/) via a Prometheus instance that is created with your YugabyteDB Anywhere installation.
```

Comment:
Slight revision to these lines:

While YugabyteDB Anywhere includes an embedded Prometheus instance and uses it for storing time-series metrics data, you can also set up your own independent and separately-managed Prometheus instance.

This additional Prometheus instance can collect, visualize, alert on, and analyze universe metrics in your own observability tools.   Prometheus itself, or any tool able to utilize Prometheus as a data source (such as Grafana) can be used.

How this works: your separate independent Prometheus instance will scrape data from DB nodes directly (in the case of VM-based universes) or scrape data from a K8s Prometheus Service Monitor (in the case of K8s-based universes).    This data scraping runs in parallel and independently with respect to YBA's own embedded Prometheus.